media-fonts/inconsolata-lgc/metadata.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>kitsunenokenja@protonmail.ch</email>
+		<name>Rob Levitsky</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>fonts@gentoo.org</email>
```

Comment:
@gentoo/fonts, you agree on co-maintaining this? If so, review appreciated.

Reply:
It's not sensible to contribute a new package and not elect myself to be its proxy maintainer, so sure I will, though I'm not sure what you mean by review here.

It's a font that hasn't changed in a few years now, so maintaining this is close to effortless.